src/coreComponents/linearAlgebra/DofManager.cpp

```diff
@@ -525,226 +666,83 @@ void DofManager::setSparsityPatternFromStencil( SparsityPatternView< globalIndex
   localIndex const numComp = field.numComponents;
   globalIndex const rankDofOffset = rankOffset();
 
-  ElementRegionManager::ElementViewAccessor< arrayView1d< globalIndex const > > dofNumber =
-    m_mesh->getElemManager().constructViewAccessor< array1d< globalIndex >, arrayView1d< globalIndex const > >( field.key );
-
-  array1d< globalIndex > rowDofIndices( numComp );
-  array1d< globalIndex > colDofIndices( numComp );
-
-  // 1. Assemble diagonal and off-diagonal blocks for elements in stencil
-  coupling.stencils->forAllStencils( *m_mesh, [&]( auto const & stencil )
+  forMeshSupport( field.support, *m_domain, [&]( MeshBody const &, MeshLevel const & mesh, auto const & regions )
   {
-    using StenciType = typename std::decay< decltype( stencil ) >::type;
-    constexpr localIndex maxNumFluxElems = StenciType::NUM_POINT_IN_FLUX;
-    constexpr localIndex maxStencilSize = StenciType::MAX_STENCIL_SIZE;
-
-    typename StenciType::IndexContainerViewConstType const & seri = stencil.getElementRegionIndices();
-    typename StenciType::IndexContainerViewConstType const & sesri = stencil.getElementSubRegionIndices();
-    typename StenciType::IndexContainerViewConstType const & sei = stencil.getElementIndices();
+    ElementRegionManager::ElementViewAccessor< arrayView1d< globalIndex const > > const dofNumber =
+      mesh.getElemManager().constructArrayViewAccessor< globalIndex, 1 >( field.key );
 
-    rowDofIndices.reserve( maxNumFluxElems );
-    colDofIndices.reserve( maxStencilSize * numComp );
+    array1d< globalIndex > rowDofIndices( numComp );
+    array1d< globalIndex > colDofIndices( numComp );
```

Comment:
Do you need to already set a size here since it will be `reserved` and `resized` again hereafter?

Reply:
No, doesn't look like it. I'm guessing this carried over accidentally from some refactoring a long time ago. Should be removed.